### PR TITLE
🥅 Guard listing NFT has been sent case

### PR DESCRIPTION
**Pull request by WeiJun0827**

### CODE REVIEW

File: src/util/api/likernft/listing.ts [hunk 1]
```diff
@@ -40,16 +40,32 @@ export function formatListingInfo(info: {
...
   return info;
 }
 
 export async function fetchNFTListingInfoByNFTId(classId: string, nftId: string) {
-  const { data } = await axios.get(`${COSMOS_LCD_INDEXER_ENDPOINT}/likechain/likenft/v1/listings/${classId}/${nftId}`);
-  const info = data.listings[0];
-  return info || null;
+  const [{ data: { listings } }, { data: { owners } }] = await Promise.all([
+    axios.get(`${COSMOS_LCD_INDEXER_ENDPOINT}/likechain/likenft/v1/listings/${classId}/${nftId}`),
+    axios.get(`${COSMOS_LCD_INDEXER_ENDPOINT}/likechain/likenft/v1/owner?class_id=${classId}`),
```
Comment (williamchong) [inline src/util/api/likernft/listing.ts]: I think there is an API that can query the owner of a class id + nft id pair without getting all owners

**Review by williamchong — CHANGES_REQUESTED** [review]

File: src/util/api/likernft/listing.ts (hunk 1 above)
Comment (WeiJun0827) [reply src/util/api/likernft/listing.ts]: Is it from [chain API](https://github.com/likecoin/likecoin-chain/blob/master/proto/likechain/likenft/v1/query.proto) or [indexer API](https://github.com/likecoin/likecoin-chain-tx-indexer/blob/master/rest/router.go)?

Comment (williamchong) [reply src/util/api/likernft/listing.ts]: from x/nft chain module
https://github.com/likecoin/likecoin-chain/blob/v3.1.1/backport/cosmos-sdk/v0.46.0-rc1/proto/cosmos/nft/v1beta1/query.proto#L19

Comment (WeiJun0827) [reply src/util/api/likernft/listing.ts]: updated, thanks!

**Review by williamchong — APPROVED** [review]

**Pull request merged**